docs/Manual_Install.md

```diff
@@ -99,44 +99,50 @@ If this machine will be a Sprout or Ralf node create the file `/etc/chronos/chro
 
 Install the Ellis package with:
 
-    sudo DEBIAN_FRONTEND=noninteractive apt-get install clearwater-management ellis --yes
+    sudo DEBIAN_FRONTEND=noninteractive apt-get install ellis --yes
+    sudo DEBIAN_FRONTEND=noninteractive apt-get install clearwater-management --yes
```

Comment:
Should we pull clearwater-management out to the end as it's common to all?  I don't feel strongly.